Sources/Model/Swift/TracksUser.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import Sentry
+
+public struct TracksUser {
```

Comment:
This is really just a moved file, but I pulled a bunch of `extension TracksUser {}` stuff out into a separate file to avoid layering problems, so now git isn't detecting the rename. Nothing changed in the body here.